### ISSUE TITLE
과제 관련 질문!

### ISSUE THREAD

**Issue by uniqueimaginate** [JonghyeokPark/SKKU_DB_PROJECT#36]
#### Description
ppt ch1 의 page 10에 있는 것을 따라해보았으나 다음과 같은 에러가 뜹니다.

#### Category
[Homework]

#### Screenshot
![Screenshot from 2019-03-25 22-24-42](https://user-images.githubusercontent.com/32162010/54923286-4a5a0380-4f4d-11e9-83af-d3c5218bb7b9.png)

**Comment by JonghyeokPark** [comment]
If you want to use the HR schema (hr account) in virtualbox vm then you must install the schema.
I attached  HR schema installation in I-CAMPUS please check it.

**Comment by Csoyee** [comment]
There is some error with schema installation guide we attached in I-CAMPUS. 
We will handle issue and re-notice how to solve the problem.

**Comment by uniqueimaginate** [comment]
Then should i wait for the next annoucement?
i can't find any annoucement in I-CAMPUS

**Comment by Csoyee** [comment]
Yes, we will notice again with correct installation guide (Maybe today or tomorrow).

Sorry for the confusion.

**Comment by uniqueimaginate** [comment]
Thank you!

**Comment by uniqueimaginate** [comment]
Another question about assignment!
As i am doing assignment by LIVE SQL, when i try to figure out Problem.B
i couldn't find employee name 'Taylor' in my HR schema.
Is this correct? I mean, in HR schema there shouldn't be an employee name 'Taylor'?

**Comment by SankyuLee** [comment]
저도 똑같은 상황을 겪었는데요.

VirtualBox의 Oracle VM에서 HR 스키마를 활성화 하시려면

`SQL> CONN SYS/oracle AS SYSDBA` (SYS계정 비밀번호는 처음 설정하신 비밀번호 입니다)
`SQL> ALTER USER HR IDENTIFIED BY HR ACCOUNT UNLOCK;`
`SQL> GRANT CONNECT, RESOURCE TO HR`
`SQL> CONN HR/HR` (ID: HR / PW: HR)

이렇게 하시면 됩니다.
ppt에 HR/OE 라고 되어있는 것은 둘 중 골라 쓰라는 의미 같습니다.
그런데 OE 계정은 없어 보입니다.

![1](https://user-images.githubusercontent.com/49195899/55403423-c6c59580-5590-11e9-863f-e860b51143c8.png)
